libraries/grpc-sdk/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@grpc/grpc-js": "^1.6.7",
     "@grpc/proto-loader": "^0.6.13",
+    "@types/fs-extra": "^9.0.13",
```

Comment:
types are dev dependencies. ioredis,lodash etc have been added in prod dependencies due to other issues. So please move.